### DISCUSSION TITLE
Tautulli Error 400

### DISCUSSION THREAD

**Discussion by bobbinz** [sct/overseerr#2662] [Help] [answered]
Hiya, Just upgraded to 1.29.0 and am trying to add Tautulli.

It keeps erroring with error 400. I've tried the internal IP, external address, with and without SSL.

```
2022-04-01T16:57:15.233Z [error][Tautulli API]: Something went wrong fetching Tautulli server info{"errorMessage":"connect ETIMEDOUT 10.10.65.4:8181"}

2022-04-01T16:57:15.233Z [error][API]: Something went wrong testing Tautulli connection{"errorMessage":"[Tautulli] Failed to fetch Tautulli server info: connect ETIMEDOUT 10.10.65.4:8181"}

{ "errorMessage": "Request failed with status code 400" }
If I go to http://ipaddresshere:8181/api/v2?apikey=HEREISMYAPIKEY&cmd=arnold it all works and gives me an Arnie quote as expected.
```

Same applies to https://properpublicaddresshere/api/v2?apikey=HEREISMYAPIKEY&cmd=arnold

So in the command line in the docker container I can reproduce the error!

```
/app # wget http://IPHERE/api/v2?apikey=KEYHERE&cmd=arnold
/app # Connecting to IPHERE
wget: server returned error: HTTP/1.1 400 Bad Request
```
However, if I jam it in inverted commas...

```
/app # wget "http://IPHERE/api/v2?apikey=KEYHERE&cmd=arnold"
Connecting to IPHERE
saving to 'v2?apikey=KEYHERE&cmd=arnold'
```
And sure enough if you cat the file there is an arnie quote sat there.

`wget http://IPHERE/api/v2?apikey=KEYHERE\&cmd=arnold` works just fine so I suspect this is to do with escaping the ampersands or something.

**Comment by TheCatLady** [comment, marked as answer]
Tautulli version? v2.9.0+ is required

**Reply by xInsertx** [reply]
I came here looking for the same issue  - maybe it is worth outlining this on the settings page (or at the minimum on the release/changelog page for others looking to update?)

otherwise, I can see people possibly raising issues once more people update (a lot of people were sitting on 2.8.1 because 2.9 had a few issues).

**Reply by Bramblefoot** [reply]
I'm on tautulli v2.10.1 and still get the same timeout as op.

**Comment by xInsertx** [comment]
I can confirm that upgrading to 2.9.5 fixed the error for me. I also have a url_base set.

I did have some weird issues with reverse proxy when viewing the pages with tautulli info but that might be my traefik setup since modifying rules slightly fixed the issue.
